src/wordpress-plugins/yoast-seo.php

```diff
@@ -320,4 +328,24 @@ protected function reset_free_installation_success_page() {
 	protected function reset_premium_installation_success_page() {
 		WPSEO_Options::set( 'activation_redirect_timestamp', '0' );
 	}
+
+	/**
+	 * Resets the First-time configuration.
+	 *
+	 * @return void
+	 */
+	protected function reset_first_time_configuration() {
+		$workouts_data                  = WPSEO_Options::get( 'workouts_data' );
+		$workouts_data['configuration'] = [ 'finishedSteps' => [] ];
+		WPSEO_Options::set( 'workouts_data', $workouts_data );
+	}
+
+	/**
+	 * Resets the Premium workouts.
+	 *
+	 * @return void
+	 */
+	protected function reset_premium_workouts() {
+		WPSEO_Options::set( 'workouts', [ 'cornerstone' => [ 'finishedSteps' => [] ] ] );
```

Comment:
Is there no need to set the `orphaned` to empty?

Reply:
This is the default value for the option, see: https://github.com/Yoast/wordpress-seo-premium/blob/27973043de3c6d05b6bf9609d37b76db44172ff7/classes/premium-option.php#L30